lib/main.js

```diff
@@ -94,22 +98,28 @@ const main = module.exports = (searchString, options, callback) => { // eslint-d
   });
 };
 
-const getFilters = main.getFilters = (searchString, callback) => { // eslint-disable-line consistent-return
+const getFilters = main.getFilters = (searchString, options, callback) => { // eslint-disable-line consistent-return
+  // Check weather no options were provided
+  if (typeof options === 'function') {
+    callback = options;
+    options = { };
+  }
+
   // Return a promise when no callback is provided
   if (!callback) {
     return new Promise((resolve, reject) => {
-      getFilters(searchString, (err, info) => { // eslint-disable-line consistent-return
-        if (err) return reject(err);
-        resolve(info);
-      });
+      getFilters(searchString, options, (err, info) => err ? reject(err) : resolve(info));
     });
   }
   if (!searchString) return callback(new Error('search string is mandatory'));
 
   let queryString;
   let parsedQuery = URL.parse(searchString, true);
-  if (parsedQuery.query.sp && parsedQuery.query.search_query) queryString = UTIL.buildFromNextpage(searchString);
-  else queryString = UTIL.buildLink(searchString);
+  if (parsedQuery.query.sp && parsedQuery.query.search_query) {
+    queryString = UTIL.buildFromNextpage(searchString, options);
+  } else {
+    queryString = UTIL.buildLink(searchString, options);
+  }
```

Comment:
tell me if i'm wrong but you're passing an object with keys "gl" and "hl" while expecting two distinct parameters "hl" and "gl"

Reply:
oh sh*t... yeah you're right, I'll fix it.